model/build.gradle

```diff
@@ -15,6 +15,8 @@ kotlin {
         android
         commonMain.dependencies {
             implementation Dep.Kotlin.bom
+            implementation Dep.Ktor.bom
+            implementation Dep.Ktor.serialization
```

Comment:
The reason I added this is because it uses the Url class !

Reply:
If you add it to the constructor of the podcast class, you may not need to depend on it in the model module. 👀

Reply:
Thanks for the review!
That's a good way. Thanks for the idea!